### ISSUE TITLE
Remove background styling from pre elements

### ISSUE THREAD

**Issue by hatched** [canonical-web-and-design/vanilla-framework#3533]
## Pattern to amend

pre

## Visual

Existing 
  https://vanillaframework.io/docs/base/code

In use 
![Screen Shot 2021-02-02 at 3 47 55 PM](https://user-images.githubusercontent.com/532033/106667069-0cc55900-656e-11eb-9f28-17db53eb7b21.png)

## Context

I often want to display user defined content while maintaining whitespace significance. Doing this within the `pre` with vanilla brings with it a number of undesired styles including the background color. It would be nice if these styles could be moved to a class added onto a `pre` instead of onto the tag directly.

**Comment by bartaz** [comment]
Removing background from `pre` may affect a lot of places that depend on current 'code block' styling of `<pre><code>` (things generated from Markdown or from discourse), so we would need to do it with caution.

But we see the point of doing that, and it seems in our recent designs we see other cases of monospace 'code-style' content (like docstrings in charmhub, etc), so if there is a need for that across the board we should consider it.

Bringing it up to WG, to discuss across the team.

**Comment by lyubomir-popov** [comment]
Agreed in the working group the correct way to handle this is to use standard `pre` / `code` styling mixed with default paragraph styles. Please let us know if this doesn't solve the problem.
